modules/ztp-post-deploy-factory-onsite-cluster.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="onsite-spoke-cluster-configuration_{context}"]
+= On site spoke cluster configuration
+
+On site customers have to do an initial configuration on the chassis to connect it to their network. The chassis comes with a pre-configured, internal, network on the eno5 interface that is used for this initial configuration process. The goal of this step is to connect eno4 to the customer’s network and configure MetalLB and other resources to match this network. Once this step is completed, the customer should switch to using the external (eno4) network instead.
+
+During the deployment the password is set to the `core user and this is used by the customer to run the final configuration steps. The user should add an `ssh` key to the nodes and remove the password from the user once the configuration steps are completed.
```

Comment:
should only `core user` be in backticks? looks like you need to add another in here to prevent the entire sentence from being included.